docs/source/modules/pyfarm.master.api.rst

```diff
@@ -14,6 +14,7 @@ Submodules
    pyfarm.master.api.pathmaps
    pyfarm.master.api.software
    pyfarm.master.api.tags
+   pyfarm.master.api.tasklogs
```

Comment:
I don't think these documentation updates should be part of this pull request. They don't even touch on the sysid vs. uuid issue.